app/controllers/requested_accounts_controller.rb

```diff
@@ -42,8 +43,18 @@ def enable_account_requests
     @course.save
   end
 
+  # List of requested accounts for a user's courses. @courses set in before action
+  def index
+    raise_unauthorized_exception unless user_signed_in? && current_user.admin?
+
+    respond_to do |format|
+      format.html { render }
+      format.json { render json: { requested_accounts: @courses.present? } }
```

Comment:
This seems like it would be doing a whole lot of unnecessary work for the json case, where all we need to a boolean. The `set_course` is going through all the current courses to find requested accounts (maybe an N+1, where it queries each course for its requests?). `RequestedAccount.any?` is probably what we want.